# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bash/mkbuiltins_have_stringize.patch.cross
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

from http://patches.openembedded.org/patch/38919/ and
http://git.yoctoproject.org/cgit/cgit.cgi/meta-mentor/commit/?id=da0ff91

authored by Christopher Larson <chris_larson@mentor.com> for yocto

fixes:
> undefined reference to `xopen'
> undefined reference to `xread'

when using FORTIFY_SOURCES during cross compilations of bash 4.2

--- ./builtins/mkbuiltins.c.orig	2013-08-22 11:01:59.845125909 +0200
+++ ./builtins/mkbuiltins.c	2013-08-22 11:03:50.113130203 +0200
@@ -28,6 +28,7 @@
 #  define HAVE_STDLIB_H
 
 #  define HAVE_RENAME
+#  define HAVE_STRINGIZE
 #endif /* CROSS_COMPILING */
 
 #if defined (HAVE_UNISTD_H)
